evals/cli/oaieval.py

```diff
@@ -50,11 +48,12 @@ def get_parser() -> argparse.ArgumentParser:
     )
     parser.add_argument("--debug", action=argparse.BooleanOptionalAction, default=False)
     parser.add_argument("--local-run", action=argparse.BooleanOptionalAction, default=True)
+    parser.add_argument("--http-run", action=argparse.BooleanOptionalAction, default=False)
+    parser.add_argument("--url", type=str, default=None)
```

Comment:
Should prefix with `http-run` in case we decide to add another url arg later on

Reply:
Changed, thank you!